code/modules/clothing/spacesuits/miscellaneous.dm

```diff
@@ -360,10 +360,6 @@ Contains:
 	actions_types = list()
 	resistance_flags = FIRE_PROOF
 
-/obj/item/clothing/suit/space/hardsuit/ert/paranormal/Initialize()
-	. = ..()
-	AddComponent(/datum/component/anti_magic, FALSE, FALSE, TRUE, ITEM_SLOT_OCLOTHING)
```

Comment:
I don't see this documented anywhere. The paranornal ERT still use paranormal ert suits even if the bezerker suit was a paranormal suit. Since your bezerker suit is no longer a paranormal hardsuit, why is this change necessary?

Reply:
the same proc appears later in the code except with arguments for everything being TRUE